spec/services/iaa_reporting_helper_spec.rb

```diff
@@ -1,8 +1,15 @@
 require 'rails_helper'
-
+RSpec.configure do |rspec|
+  rspec.expect_with :rspec do |c|
+    c.max_formatted_output_length = nil
+  end
+end
```

Comment:
I am guessing we do not want to keep this
```suggestion

```

Reply:
yes, not needed ,added for debugging.